.github/workflows/dockers_builder_dev.yml

```diff
@@ -9,7 +9,6 @@ env:
   jessie_version: debian8
   bullseye_version: debian11
   front_jessie_navitia_components: 'tyr-web instances-configurator'
-  front_bullseye_navitia_components: 'jormungandr'
```

Comment:
Why do you remove this one?

Reply:
Unused variable

Reply:
Ah, it's built in this workflow but not managed exactly the same way as other jobs.